Sources/DolbyIORTSCore/StreamOrchestrator.swift

```diff
@@ -115,9 +117,20 @@ open class StreamOrchestrator {
                     }
                 }
                 
+                // Unproject audio whose source has been stopped from the publisher
+                if let lastSource = self.lastSelectedAudioSource,
+                   lastSource.sourceId != source.sourceId,
+                   sources.first(where: { $0.id == lastSource.id }) == nil {
+                    group.addTask {
+                        await self.stateMachine.setPlayingAudio(false, for: lastSource)
+                        self.subscriptionManager.unprojectAudio(for: lastSource)
+                    }
+                }
```

Comment:
When a source gets removed by the publisher, is the projected audio valid at all ?

Reply:
At least the unproject() works. Though the publisher stopped a stream, the SDK internally still keeps a reference to it. Therefore, if the same source ID is published again. the audio track played. This is the issue reported by the QA.

Reply:
I suggest its worth fixing it in the media server or the SDK itself, that the existing projections for both audio and video should be removed when the publisher stops publishing or removes a source ?